examples/hello_world/python_non_petastorm_hello_world.py

```diff
@@ -0,0 +1,32 @@
+#  Copyright (c) 2017-2018 Uber Technologies, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Minimal example of how to read samples from a dataset generated by `generate_non_petastorm_dataset.py`
+using plain Python"""
+
+from __future__ import print_function
+
+from petastorm import make_batch_reader
+
+
+def python_non_petastorm_hello_world(dataset_url='file:///tmp/non_petastorm_dataset'):
+    # Reading data from the non-Petastorm Parquet file with inferred Unischema
+    with make_batch_reader(dataset_url, schema_fields=["id", "value1", "value2", "value3", "value4"]) as reader:
+        schema_view = reader.next()
```

Comment:
@selitvin Is there a `correct` way to get column data from `Reader` other than by calling `.next()`? This feels wrong.

Reply:
A `Reader` is essentially a python iterator so the correct way to get data from it would be to iterate through it or use `next(reader)` for a single row